specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/keys.json

```diff
@@ -1488,7 +1489,7 @@
         },
         "exportable": {
           "type": "boolean",
-          "description": "Indicates if the private key can be exported."
+          "description": "Indicates if the private key can be exported. Release policy must be provided when creating the 1st version of an exportable key."
```

Comment:
Nit: "1st" doesn't localized well, if this gets localized at all (not sure). I recommend "first".

Reply:
Resolved